src/Twig/StorageExtension.php

```diff
@@ -26,6 +30,10 @@ public function getFilters(): array
 
     public function formatUrl(string $path): string
     {
-        return $this->awsS3PublicUrlResolver->resolve($path);
+        if (preg_match(self::AUTHORIZED_SCHEMA_REGEX, $path)) {
+            return $this->assetsHelper->getUrl('/images/default_speaker.svg');
```

Comment:
The `$path` is already the full absolute path here, so just `return $path;`

Reply:
No here $path is not always full absolute path. Its the name File and its formatUrl function to apply the rest of the url. And thomas say me to apply the development here.

Reply:
I said here, meaning in the if condition...